api/app/events/models.py

```diff
@@ -47,7 +47,7 @@ class Event(db.Model):
     offer_close = db.Column(db.DateTime(), nullable=True)
     registration_open = db.Column(db.DateTime(), nullable=True)
     registration_close = db.Column(db.DateTime(), nullable=True)
-    event_type = db.Column(db.Enum(EventType), nullable=False)
+    event_type = db.Column(db.Enum(EventType, name="event_type"), nullable=False)
```

Comment:
alembic is trying to rename the deb enum to `eventtype` in the migrations for some reason. This prevents that.